### ISSUE TITLE
Navbar and footer enhancement (UI enhancement)

### ISSUE THREAD

**Issue by bhutani2002** [MetaVASP/Zipify#25]
As we are fulfilling the purpose of zipping or compressing the file and the main part is present on the center of the page, the page doesn't required to be scrolling as it bothers the ui experience and for that footer image should be removed. Or else if scrolling is okay than the header should be made sticky and not movable.

Are you working on the issue?
Yes

**Comment by bhutani2002** [comment]
Hey, @pulkit-30 could you please review this issue and provide me your feedback. It's just my opinion from user point of view, and am ready to fix it if you allow.
Thank You.

**Comment by pulkit-30** [comment]
Yep!  Agreed , page is not required to scroll 👍🏽 ,
May you please suggest something better than removing footer image

**Comment by bhutani2002** [comment]
> Yep! Agreed , page is not required to scroll 👍🏽 , May you please suggest something better than removing footer image

Yep I considered other option like reducing height and width of footer image and it too didn't removed the scrolling functionality. Since this is 1 webpage website, we can reduce every divs or boxes dimensions(headers, main box and footer image) without removing the footer. 

What do you think?
I'll do that if you suggest

**Comment by Abhilasha-Jairath** [comment]
I want to work on this under GSSOC2022. Can you assign it to me.

**Comment by bhutani2002** [comment]
> I want to work on this under GSSOC2022. Can you assign it to me.

Hey, @Abhilasha-Jairath we are working on this, as it's mentioned on the issue itself too. You can find other issues and can create yours too.
Thank You

**Comment by Abhilasha-Jairath** [comment]
> > I want to work on this under GSSOC2022. Can you assign it to me.
> 
> Hey, @Abhilasha-Jairath we are working on this, as it's mentioned on the issue itself too. You can find other issues and can create yours too. Thank You

ok !! thank you

**Comment by bhutani2002** [comment]
> > Yep! Agreed , page is not required to scroll 👍🏽 , May you please suggest something better than removing footer image
> 
> Yep I considered other option like reducing height and width of footer image and it too didn't removed the scrolling functionality. Since this is 1 webpage website, we can reduce every div or box dimensions(headers, main box and footer image) without removing the footer.
> 
> What do you think? I'll do that if you suggest

Hey @pulkit-30 , any feedback?

**Comment by pulkit-30** [comment]
> Yep I considered other option like reducing height and width of footer image and it too didn't removed the scrolling functionality. Since this is 1 webpage website, we can reduce every div or box dimensions(headers, main box and footer image) without removing the footer.
> 
> What do you think? I'll do that if you suggest

Let's wait for UI improvement after that we can work on this 

Issue #1 
PR #20

**Comment by bhutani2002** [comment]
> > Yep I considered other option like reducing height and width of footer image and it too didn't removed the scrolling functionality. Since this is 1 webpage website, we can reduce every div or box dimensions(headers, main box and footer image) without removing the footer.
> > What do you think? I'll do that if you suggest
> 
> Let's wait for UI improvement after that we can work on this
> 
> Issue #1 PR #20

Hmm let's wait till then

**Comment by TANYA-CHAN** [comment]
I want to work on this under GSSOC'22, could you please assign it to me?

**Comment by bhutani2002** [comment]
> I want to work on this under GSSOC'22, could you please assign it to me?

Hey @TANYA-CHAN this issue is under evaluation and has to be think upon. So, you can check other issues.
Thank You
